crates/router/src/db/api_keys.rs

```diff
@@ -67,10 +69,46 @@ impl ApiKeyInterface for Store {
         api_key: storage::ApiKeyUpdate,
     ) -> CustomResult<storage::ApiKey, errors::StorageError> {
         let conn = connection::pg_connection_write(self).await?;
-        storage::ApiKey::update_by_merchant_id_key_id(&conn, merchant_id, key_id, api_key)
+        let _merchant_id = merchant_id.clone();
```

Comment:
So these variables are only used in `accounts_cache` feature. Otherwise these variables are unused. So have to declare with underscore prefix for warnings